src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/ProjectsController.java

```diff
@@ -162,6 +162,23 @@ public String getSynchronizeProjectPage() {
 		return SYNC_NEW_PROJECT_PAGE;
 	}
 
+	/**
+	 * Get the page to share samples between projects
+	 *
+	 * @param projectId Identifier for the current project
+	 * @param model     Spring model for template variables
+	 * @param principal Currently logged in user
+	 * @return Path to the template for shareing samples
+	 */
+	@RequestMapping("/projects/{projectId}/share")
+	public String getProjectsSharePage(@PathVariable Long projectId, final Model model, final Principal principal) {
+		Project project = projectService.read(projectId);
+		model.addAttribute("project", project);
```

Comment:
I think you can remove this line as the code executed by the next line adds the project to the template

Reply:
Oops, bad copy and paste :) Removed